jbehave-core/src/main/java/org/jbehave/core/model/ExamplesTable.java

```diff
@@ -94,6 +94,29 @@
  * | value 11#comment | value 12 | .... | value 1n |
  * </pre>
  * Comments including the separator are stripped.</p>
+ *
+ * <p>Line break is a default separator for rows in ExamplesTable, that's why they can't be added as is
+ * to the data. In order to put the value with line breaks to ExamplesTable escape sequences (a character preceded by a
+ * backslash (\) is an escape sequence) must be used.
+ * <table>
+ *  <thead><tr><th>Escape Sequence</th><th>Description</th></tr></thead>
+ *  <tbody>
+ *   <tr><td>\n</td><td>Insert a newline in the value at this point.</td></tr>
+ *   <tr><td>\r</td><td>Insert a carriage return in the text at this point.</td></tr>
+ *   <tr><td>\\</td><td>Insert a backslash character in the text at this point.</td></tr>
+ *  </tbody>
+ * </table>
+ * Inlined property <code>processEscapeSequences</code> defines whether escape sequences should be replaced in the data.
+ * It’s <code>false</code> by default (no property is declared explicitly). The allowed values are <code>true</code> and
+ * <code>false</code>, any other values are considered invalid and will lead to exception thrown at parsing.
+ * <pre>
+ * {processEscapeSequences=true, commentSeparator=#}
+ * |header          |
+ * |line 1\nline 2  |# The value with a newline
+ * |line 1\r\nline 2|# The value with a carriage return and a newline
+ * |line 1\\nline 2 |# The value with an escaped escape sequence, the result will be "line 1\nline 2"
```

Comment:
will it be convenient if we want to alternate between unix \n and msdos \n\r while test runs?